Source/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
 
   <!-- Target framework -->
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Why can't we have `net6.0`?

Reply:
Ooops good catch. I have had problems with a merge commit that I had to resolve manually, and it looks like this was one of the errors that I missed.